.travis.yml

```diff
@@ -8,10 +8,9 @@ matrix:
 
     # Python 3 + QT5
     - os: linux
-      dist: trusty
```

Comment:
may I know why we should change?

Reply:
Trusty reached end of standard support two years ago.
https://wiki.ubuntu.com/Releases